--- wmint-0.9/wmint/Makefile.orig	2000-01-08 21:25:18.000000000 +0100
+++ wmint-0.9/wmint/Makefile	2010-01-03 01:25:43.486099334 +0100
@@ -1,23 +1,22 @@
-LIBDIR = -L/usr/X11R6/lib
 LIBS   = -lXpm -lX11 -lXext
-FLAGS = -O2 -Wall 
+CFLAGS = -O2 -Wall 
 OBJS =	wmint.o ../wmgeneral/wmgeneral.o 
 #		../wmgeneral/misc.o \
 #                ../wmgeneral/list.o
 
 .c.o:
-	cc -I/usr/X11R6/share/include $(FLAGS) -c -Wall $< -o $*.o
+	$(CC) $(CFLAGS) -c -Wall $< -o $*.o
 
 wmint: $(OBJS)
-	cc $(FLAGS) -o wmint $^ $(LIBDIR) $(LIBS)
+	$(CC) $(CFLAGS) $(LDFLAGS) -o wmint $^ $(LIBS)
 
 all:: wmint
 
 clean::
 	for i in $(OBJS) ; do \
-		rm $$i ; \
+		rm -f $$i ; \
 	done
-	rm wmint
+	rm -f wmint
 
 install:: wmint
 	cp -f wmint /usr/local/bin/
